pkg/syncer/util.go

```diff
@@ -17,6 +18,11 @@ import (
 	csitypes "sigs.k8s.io/vsphere-csi-driver/pkg/csi/types"
 )
 
+const (
+	// PVC annotation key to specify storage class from which PV should be provisioned
+	scNameAnnotationKey = "volume.beta.kubernetes.io/storage-class"
```

Comment:
Based on the documentation - https://kubernetes.io/docs/concepts/storage/persistent-volumes/

> In the past, the annotation volume.beta.kubernetes.io/storage-class was used instead of the storageClassName attribute. This annotation is still working; however, it will become fully deprecated in a future Kubernetes release.

 Can we make sure not to introduce something which is becoming deprecated in the future Kubernetes release?

Reply:
Explained in the above comment https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/303#discussion_r468564958.